services/api/src/resources/deployment/resolvers.ts

```diff
@@ -181,13 +181,29 @@ export const getDeploymentsByFilter: ResolverFn = async (
       queryBuilder = queryBuilder.whereIn('environment.project', userProjectIds);
   }
 
+  // collect builds for a specific year/month
+  if (month) {
```

Comment:
perhaps this should be renamed, given we're using the year and month for the filter

Reply:
I've updated it now to be `startDate` and `endDate` filters